deluge/common.py

```diff
@@ -44,6 +45,13 @@
 
     os.environ['SSL_CERT_FILE'] = where()
 
+# On windows sock.if_nameindex() is only supported on 3.8+ and does not return a uuid.
+# Try ifaddr to verify network interfaces.
+try:
+    import ifaddr
```

Comment:
We need to mention the optional dependency in DEPENDS.md and add it setup.py extras